### PR TITLE
Fixed +beta

### PR DESCRIPTION
**Pull request by 121jwang**
All the file paths might have issues

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -103,17 +103,14 @@
...
-                # switch it to the state it isn't (toggle it)
-                if toggle_state:
-                    toggle_state = False
-                else:
-                    toggle_state = True
-            else:
-                # doesnt have perms
-                await client.send_message(message.channel, "**Sorry, but you do not have the manage server permission, have somebody with it use this command instead!**")
+        state = servers[str(message.server.id)]["beta_features"]
+        if False:
```
Comment (RDIL) [inline main.py]: This will never run..

Comment (121jwang) [reply main.py]: Yeah I know. I used it for testing because your permissions were giving errors

Comment (RDIL) [reply main.py]: so should we fix this?

Comment (121jwang) [reply main.py]: if message.author.permissions.manage_server:
This line gives an error

Comment (121jwang) [reply main.py]: So I think that's the permission section on your end, so I just it to false for now

**Pull request merged**